src/main/java/ch/heigvd/pro/b04/polls/exceptions/IllegalPollStateException.java

```diff
@@ -0,0 +1,7 @@
+package ch.heigvd.pro.b04.polls.exceptions;
+
+public class IllegalPollStateException extends IllegalStateException {
```

Comment:
IllegalPollStateException is not checked, but we probably want to make it checked to enforce `try ... catch ...` blocks around methods that might throw it.